Building with "Unix Makefiles" as CMake generator no longer works

Building with CMake directly gives errors:

$ cmake -Bbuild -DBOARD=xmc47_relax_kit samples/hello_world/
$ make -Cbuild

100%] Linking C executable zephyr.elf
/home/andriy/zephyr-sdk-0.16.5/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd: cannot open linker script file zephyr/isr_tables_swi.ld: No such file or directory
collect2: error: ld returned 1 exit status

While building with west works fine:
$ west build -p -b xmc47_relax_kit samples/hello_world/

This issue is most likely caused by PR: 
https://github.com/zephyrproject-rtos/zephyr/pull/66392

For me, building with `west` doesn't work either after https://github.com/zephyrproject-rtos/zephyr/commit/1ff24b34ebe568a5d606928af69914de946b9762:

```
$ west build -b frdm_k64f samples/hello_world/ 2>&1 | tail -n 7
[ 97%] Linking C executable zephyr.elf
/opt/zephyr-sdk/0.16.5/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd: cannot open linker script file zephyr/isr_tables_swi.ld: No such file or directory
collect2: error: ld returned 1 exit status
gmake[2]: *** [zephyr/CMakeFiles/zephyr_final.dir/build.make:143: zephyr/zephyr.elf] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:3216: zephyr/CMakeFiles/zephyr_final.dir/all] Error 2
gmake: *** [Makefile:91: all] Error 2
FATAL ERROR: command exited with status 2: /home/brix/Projects/zephyrproject/venv/bin/cmake --build /home/brix/Projects/zephyrproject/zephyr/build

```

@rakons Full build log:

```$ west build -p always -b frdm_k64f samples/hello_world/
-- west build: making build dir /home/hebad/Projects/zephyrproject/zephyr/build pristine
-- west build: generating a build system
Loading Zephyr default modules (Zephyr base).
-- Application: /home/hebad/Projects/zephyrproject/zephyr/samples/hello_world
-- CMake version: 3.27.7
-- Found Python3: /home/hebad/Projects/zephyrproject/venv/bin/python3 (found suitable version "3.10.12", minimum required is "3.8") found components: Interpreter 
-- Cache files will be written to: /home/hebad/.cache/zephyr
-- Zephyr version: 3.5.99 (/home/hebad/Projects/zephyrproject/zephyr)
-- Found west (found suitable version "1.2.0", minimum required is "0.14.0")
-- Board: frdm_k64f
-- ZEPHYR_TOOLCHAIN_VARIANT not set, trying to locate Zephyr SDK
-- Found host-tools: zephyr 0.16.5 (/opt/zephyr-sdk/0.16.5)
-- Found toolchain: zephyr 0.16.5 (/opt/zephyr-sdk/0.16.5)
-- Found Dtc: /opt/zephyr-sdk/0.16.5/sysroots/x86_64-pokysdk-linux/usr/bin/dtc (found suitable version "1.6.0", minimum required is "1.4.6") 
-- Found BOARD.dts: /home/hebad/Projects/zephyrproject/zephyr/boards/arm/frdm_k64f/frdm_k64f.dts
-- Generated zephyr.dts: /home/hebad/Projects/zephyrproject/zephyr/build/zephyr/zephyr.dts
-- Generated devicetree_generated.h: /home/hebad/Projects/zephyrproject/zephyr/build/zephyr/include/generated/devicetree_generated.h
-- Including generated dts.cmake file: /home/hebad/Projects/zephyrproject/zephyr/build/zephyr/dts.cmake
Parsing /home/hebad/Projects/zephyrproject/zephyr/Kconfig
Loaded configuration '/home/hebad/Projects/zephyrproject/zephyr/boards/arm/frdm_k64f/frdm_k64f_defconfig'
Merged configuration '/home/hebad/Projects/zephyrproject/zephyr/samples/hello_world/prj.conf'
Configuration saved to '/home/hebad/Projects/zephyrproject/zephyr/build/zephyr/.config'
Kconfig header saved to '/home/hebad/Projects/zephyrproject/zephyr/build/zephyr/include/generated/autoconf.h'
-- Found GnuLd: /opt/zephyr-sdk/0.16.5/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd (found version "2.38") 
-- The C compiler identification is GNU 12.2.0
-- The CXX compiler identification is GNU 12.2.0
-- The ASM compiler identification is GNU
-- Found assembler: /opt/zephyr-sdk/0.16.5/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc
Load components for MK64F12:
driver_common component is included.
driver_reset component is included.
device_CMSIS component is included.
CMSIS_Include_core_cm component is included.
device_system component is included.
driver_uart component is included.
driver_port component is included.
driver_sysmpu component is included.
-- Using ccache: /usr/bin/ccache
CMake Warning at /home/hebad/Projects/zephyrproject/zephyr/CMakeLists.txt:864 (message):
  No SOURCES given to Zephyr library: lib__posix__options

  Excluding target from build.


-- Configuring done (20.2s)
-- Generating done (0.1s)
-- Build files have been written to: /home/hebad/Projects/zephyrproject/zephyr/build
-- west build: building application
[  0%] Generating misc/generated/syscalls.json, misc/generated/struct_tags.json
[  2%] Built target parse_syscalls_target
[  3%] Generating include/generated/kobj-types-enum.h, include/generated/otype-to-str.h, include/generated/otype-to-size.h
[  3%] Built target kobj_types_h_target
[  4%] Generating include/generated/syscall_dispatch.c, include/generated/syscall_list.h
[  4%] Built target syscall_list_h_target
[  5%] Generating include/generated/driver-validation.h
[  5%] Built target driver_validation_h_target
[  6%] Building C object zephyr/CMakeFiles/offsets.dir/arch/arm/core/offsets/offsets.c.obj
[  6%] Built target offsets
[  7%] Generating include/generated/offsets.h
[  7%] Built target offsets_h
[  7%] Generating include/generated/version.h
-- Zephyr version: 3.5.99 (/home/hebad/Projects/zephyrproject/zephyr), build: zephyr-v3.5.0-5374-g826d67af9169
[  7%] Built target version_h
[  7%] Built target zephyr_generated_headers
[  8%] Building C object CMakeFiles/app.dir/src/main.c.obj
[  8%] Linking C static library app/libapp.a
[  8%] Built target app
[  9%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/heap/heap.c.obj
[ 10%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/cbprintf_packaged.c.obj
[ 11%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/printk.c.obj
[ 11%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/sem.c.obj
[ 12%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/thread_entry.c.obj
[ 13%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/cbprintf_complete.c.obj
[ 15%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/assert.c.obj
[ 16%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/utils/dec.c.obj
[ 16%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/utils/hex.c.obj
[ 17%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/utils/rb.c.obj
[ 18%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/utils/timeutil.c.obj
[ 19%] Building C object zephyr/CMakeFiles/zephyr.dir/lib/utils/bitarray.c.obj
[ 19%] Building C object zephyr/CMakeFiles/zephyr.dir/misc/generated/configs.c.obj
[ 20%] Building C object zephyr/CMakeFiles/zephyr.dir/soc/arm/nxp_kinetis/flash_configuration.c.obj
[ 21%] Building C object zephyr/CMakeFiles/zephyr.dir/soc/arm/nxp_kinetis/k6x/soc.c.obj
[ 22%] Building C object zephyr/CMakeFiles/zephyr.dir/soc/arm/nxp_kinetis/k6x/nxp_mpu_regions.c.obj
[ 22%] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/mem_mgmt/mem_attr.c.obj
[ 23%] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/tracing/tracing_none.c.obj
[ 24%] Linking C static library libzephyr.a
[ 24%] Built target zephyr
[ 25%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/main_weak.c.obj
[ 26%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/banner.c.obj
[ 26%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/busy_wait.c.obj
[ 27%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/device.c.obj
[ 29%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/errno.c.obj
[ 30%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/fatal.c.obj
[ 30%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/init.c.obj
[ 31%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/kheap.c.obj
[ 32%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/mem_slab.c.obj
[ 33%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/thread.c.obj
[ 33%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/version.c.obj
[ 34%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/sched.c.obj
[ 35%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/idle.c.obj
[ 36%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/mailbox.c.obj
[ 37%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/msg_q.c.obj
[ 37%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/mutex.c.obj
[ 38%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/queue.c.obj
[ 39%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/sem.c.obj
[ 40%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/stack.c.obj
[ 40%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/system_work_q.c.obj
[ 41%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/work.c.obj
[ 43%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/condvar.c.obj
[ 44%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/xip.c.obj
[ 44%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/timeout.c.obj
[ 45%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/timer.c.obj
[ 46%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/mempool.c.obj
[ 47%] Building C object zephyr/kernel/CMakeFiles/kernel.dir/dynamic_disabled.c.obj
[ 47%] Linking C static library libkernel.a
[ 47%] Built target kernel
[ 48%] Generating linker_zephyr_pre0.cmd
[ 48%] Built target linker_zephyr_prebuilt_script_target
[ 48%] Building C object zephyr/arch/common/CMakeFiles/isr_tables.dir/isr_tables.c.obj
[ 49%] Linking C static library libisr_tables.a
[ 49%] Built target isr_tables
[ 50%] Building C object zephyr/arch/common/CMakeFiles/arch__common.dir/sw_isr_common.c.obj
[ 51%] Linking C static library libarch__common.a
[ 51%] Built target arch__common
[ 52%] Building C object zephyr/arch/arch/arm/core/CMakeFiles/arch__arm__core.dir/fatal.c.obj
[ 53%] Building C object zephyr/arch/arch/arm/core/CMakeFiles/arch__arm__core.dir/nmi.c.obj
[ 54%] Building ASM object zephyr/arch/arch/arm/core/CMakeFiles/arch__arm__core.dir/nmi_on_reset.S.obj
[ 54%] Building C object zephyr/arch/arch/arm/core/CMakeFiles/arch__arm__core.dir/tls.c.obj
[ 55%] Linking C static library libarch__arm__core.a
[ 55%] Built target arch__arm__core
[ 56%] Building ASM object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/exc_exit.S.obj
[ 58%] Building C object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/fault.c.obj
[ 58%] Building ASM object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/fault_s.S.obj
[ 59%] Building C object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/fpu.c.obj
[ 60%] Building ASM object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/reset.S.obj
[ 61%] Building C object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/scb.c.obj
[ 62%] Building C object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/thread_abort.c.obj
[ 62%] Building ASM object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/vector_table.S.obj
[ 63%] Building C object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/swap.c.obj
[ 64%] Building ASM object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/swap_helper.S.obj
[ 65%] Building C object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/irq_manage.c.obj
[ 65%] Building C object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/prep_c.c.obj
[ 66%] Building C object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/thread.c.obj
[ 67%] Building ASM object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/cpu_idle.S.obj
[ 68%] Building C object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/irq_init.c.obj
[ 68%] Building ASM object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/isr_wrapper.S.obj
[ 69%] Building ASM object zephyr/arch/arch/arm/core/cortex_m/CMakeFiles/arch__arm__core__cortex_m.dir/__aeabi_read_tp.S.obj
[ 70%] Linking C static library libarch__arm__core__cortex_m.a
[ 70%] Built target arch__arm__core__cortex_m
[ 72%] Building C object zephyr/arch/arch/arm/core/mpu/CMakeFiles/arch__arm__core__mpu.dir/arm_core_mpu.c.obj
[ 72%] Building C object zephyr/arch/arch/arm/core/mpu/CMakeFiles/arch__arm__core__mpu.dir/nxp_mpu.c.obj
[ 73%] Linking C static library libarch__arm__core__mpu.a
[ 73%] Built target arch__arm__core__mpu
[ 74%] Building C object zephyr/lib/libc/picolibc/CMakeFiles/lib__libc__picolibc.dir/libc-hooks.c.obj
[ 75%] Linking C static library liblib__libc__picolibc.a
[ 75%] Built target lib__libc__picolibc
[ 76%] Building C object zephyr/lib/libc/common/CMakeFiles/lib__libc__common.dir/source/stdlib/abort.c.obj
[ 77%] Building C object zephyr/lib/libc/common/CMakeFiles/lib__libc__common.dir/source/stdlib/malloc.c.obj
[ 77%] Linking C static library liblib__libc__common.a
[ 77%] Built target lib__libc__common
[ 78%] Building C object zephyr/drivers/clock_control/CMakeFiles/drivers__clock_control.dir/clock_control_mcux_mcg.c.obj
[ 78%] Building C object zephyr/drivers/clock_control/CMakeFiles/drivers__clock_control.dir/clock_control_mcux_sim.c.obj
[ 79%] Linking C static library libdrivers__clock_control.a
[ 79%] Built target drivers__clock_control
[ 80%] Building C object zephyr/drivers/console/CMakeFiles/drivers__console.dir/uart_console.c.obj
[ 81%] Linking C static library libdrivers__console.a
[ 81%] Built target drivers__console
[ 81%] Building C object zephyr/drivers/gpio/CMakeFiles/drivers__gpio.dir/gpio_mcux.c.obj
[ 82%] Linking C static library libdrivers__gpio.a
[ 82%] Built target drivers__gpio
[ 82%] Building C object zephyr/drivers/pinctrl/CMakeFiles/drivers__pinctrl.dir/common.c.obj
[ 83%] Building C object zephyr/drivers/pinctrl/CMakeFiles/drivers__pinctrl.dir/pinctrl_kinetis.c.obj
[ 84%] Linking C static library libdrivers__pinctrl.a
[ 84%] Built target drivers__pinctrl
[ 86%] Building C object zephyr/drivers/serial/CMakeFiles/drivers__serial.dir/uart_mcux.c.obj
[ 86%] Linking C static library libdrivers__serial.a
[ 86%] Built target drivers__serial
[ 87%] Building C object zephyr/drivers/timer/CMakeFiles/drivers__timer.dir/sys_clock_init.c.obj
[ 88%] Building C object zephyr/drivers/timer/CMakeFiles/drivers__timer.dir/cortex_m_systick.c.obj
[ 89%] Linking C static library libdrivers__timer.a
[ 89%] Built target drivers__timer
[ 89%] Building C object modules/hal_nxp/hal_nxp/CMakeFiles/..__modules__hal__nxp.dir/mcux/mcux-sdk/devices/MK64F12/drivers/fsl_clock.c.obj
[ 90%] Building C object modules/hal_nxp/hal_nxp/CMakeFiles/..__modules__hal__nxp.dir/mcux/mcux-sdk/drivers/common/fsl_common.c.obj
[ 91%] Building C object modules/hal_nxp/hal_nxp/CMakeFiles/..__modules__hal__nxp.dir/mcux/mcux-sdk/drivers/common/fsl_common_arm.c.obj
[ 92%] Building C object modules/hal_nxp/hal_nxp/CMakeFiles/..__modules__hal__nxp.dir/mcux/mcux-sdk/devices/MK64F12/system_MK64F12.c.obj
[ 92%] Building C object modules/hal_nxp/hal_nxp/CMakeFiles/..__modules__hal__nxp.dir/mcux/mcux-sdk/drivers/uart/fsl_uart.c.obj
[ 93%] Building C object modules/hal_nxp/hal_nxp/CMakeFiles/..__modules__hal__nxp.dir/mcux/mcux-sdk/drivers/sysmpu/fsl_sysmpu.c.obj
[ 94%] Linking C static library lib..__modules__hal__nxp.a
[ 94%] Built target ..__modules__hal__nxp
[ 95%] Building C object zephyr/CMakeFiles/zephyr_pre0.dir/misc/empty_file.c.obj
[ 96%] Linking C executable zephyr_pre0.elf
Logical command for additional byproducts on target: zephyr_pre0
[ 96%] Built target zephyr_pre0
[ 96%] Generating linker.cmd
[ 96%] Built target linker_zephyr_final_script_target
[ 97%] Generating isr_tables.c, isr_tables_vt.ld, isr_tables_swi.ld
[ 97%] Building C object zephyr/CMakeFiles/zephyr_final.dir/misc/empty_file.c.obj
[ 98%] Building C object zephyr/CMakeFiles/zephyr_final.dir/isr_tables.c.obj
[100%] Linking C executable zephyr.elf
/opt/zephyr-sdk/0.16.5/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.2.0/../../../../arm-zephyr-eabi/bin/ld.bfd: cannot open linker script file zephyr/isr_tables_swi.ld: No such file or directory
collect2: error: ld returned 1 exit status
gmake[2]: *** [zephyr/CMakeFiles/zephyr_final.dir/build.make:143: zephyr/zephyr.elf] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:3215: zephyr/CMakeFiles/zephyr_final.dir/all] Error 2
gmake: *** [Makefile:91: all] Error 2
FATAL ERROR: command exited with status 2: /home/hebad/Projects/zephyrproject/venv/bin/cmake --build /home/hebad/Projects/zephyrproject/zephyr/build```

--- a/arch/arm/core/swi_tables.ld
+++ b/arch/arm/core/swi_tables.ld
@@ -4,5 +4,5 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 #if LINKER_ZEPHYR_FINAL && defined(CONFIG_ISR_TABLES_LOCAL_DECLARATION)
-INCLUDE zephyr/isr_tables_swi.ld
+INCLUDE isr_tables_swi.ld
 #endif

--- a/arch/arm/core/swi_tables.ld
+++ b/arch/arm/core/swi_tables.ld
@@ -3,6 +3,6 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-#if LINKER_ZEPHYR_FINAL
+#if LINKER_ZEPHYR_FINAL && defined(CONFIG_ISR_TABLES_LOCAL_DECLARATION)
 INCLUDE zephyr/isr_tables_swi.ld
 #endif

--- a/arch/arm/core/vector_table.ld
+++ b/arch/arm/core/vector_table.ld
@@ -51,7 +51,7 @@ _vector_start = .;
 KEEP(*(.exc_vector_table))
 KEEP(*(".exc_vector_table.*"))
 
-#if LINKER_ZEPHYR_FINAL && CONFIG_ISR_TABLES_LOCAL_DECLARATION
+#if LINKER_ZEPHYR_FINAL && defined(CONFIG_ISR_TABLES_LOCAL_DECLARATION)
 INCLUDE zephyr/isr_tables_vt.ld
 #else
 KEEP(*(.vectors))

--- a/arch/arm/core/vector_table.ld
+++ b/arch/arm/core/vector_table.ld
@@ -52,7 +52,7 @@ KEEP(*(.exc_vector_table))
 KEEP(*(".exc_vector_table.*"))
 
 #if LINKER_ZEPHYR_FINAL && defined(CONFIG_ISR_TABLES_LOCAL_DECLARATION)
-INCLUDE zephyr/isr_tables_vt.ld
+INCLUDE isr_tables_vt.ld
 #else
 KEEP(*(.vectors))
 #endif

--- a/arch/arm64/core/swi_tables.ld
+++ b/arch/arm64/core/swi_tables.ld
@@ -4,5 +4,5 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 #if LINKER_ZEPHYR_FINAL && defined(CONFIG_ISR_TABLES_LOCAL_DECLARATION)
-INCLUDE zephyr/isr_tables_swi.ld
+INCLUDE isr_tables_swi.ld
 #endif

--- a/arch/arm64/core/swi_tables.ld
+++ b/arch/arm64/core/swi_tables.ld
@@ -3,6 +3,6 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-#if LINKER_ZEPHYR_FINAL
+#if LINKER_ZEPHYR_FINAL && defined(CONFIG_ISR_TABLES_LOCAL_DECLARATION)
 INCLUDE zephyr/isr_tables_swi.ld
 #endif

--- a/include/zephyr/arch/arm64/scripts/linker.ld
+++ b/include/zephyr/arch/arm64/scripts/linker.ld
@@ -112,7 +112,7 @@ SECTIONS
         KEEP(*(".exc_vector_table.*"))
 
 #if LINKER_ZEPHYR_FINAL && defined(CONFIG_ISR_TABLES_LOCAL_DECLARATION)
-        INCLUDE zephyr/isr_tables_vt.ld
+        INCLUDE isr_tables_vt.ld
 #else
         KEEP(*(.vectors))
 #endif

--- a/include/zephyr/arch/arm64/scripts/linker.ld
+++ b/include/zephyr/arch/arm64/scripts/linker.ld
@@ -111,7 +111,7 @@ SECTIONS
         KEEP(*(.exc_vector_table))
         KEEP(*(".exc_vector_table.*"))
 
-#if LINKER_ZEPHYR_FINAL && CONFIG_ISR_TABLES_LOCAL_DECLARATION
+#if LINKER_ZEPHYR_FINAL && defined(CONFIG_ISR_TABLES_LOCAL_DECLARATION)
         INCLUDE zephyr/isr_tables_vt.ld
 #else
         KEEP(*(.vectors))

